BranchMetrics/ios-branch-deep-linking-attribution
# SDK-1741 fix for skan error handling on iOS 15.4

## Reference
SDK-1741 Fix for SKAN error handling on iOS 15.4

## Summary
There was an inadvertent return preventing callbacks on iOS 15.4. 

## Motivation
Fix callback issue on iOS 15.4. T

## Type Of Change

## Testing Instructions
Using a test app enabled for SKAN, attempt to open a deeplink on iOS 15.4 with SDK 1.45.1. The early return prevents the callback from getting fired.

Using the same test app, attempt to open a deeplink on iOS 15.4 with SDK 1.45.2. Callback should be fired correctly.









cc @BranchMetrics/saas-sdk-devs for visibility.
